test/e2e/certificate.spec.ts

```diff
@@ -14,21 +14,41 @@ test.describe('Certificate Page', () => {
 
     await expect(domainName).toContainText('user1.starchart.com');
     await expect(titleHeader).toContainText('Certificate');
+
     await page.getByRole('button', { name: 'Request a Certificate' }).click();
 
-    page
+    const loadingPageText = page
       .locator('div')
       .filter({
         hasText: 'We have received your request, and will notify you when your certificate is read',
       })
       .nth(1);
 
+    await expect(loadingPageText).toContainText(
+      'We have received your request, and will notify you when your certificate is read'
+    );
```

Comment:
Would it be better to add a `class` or `id` to this div, and use that to select the div? Here, we are selecting by text, and then checking the same text, kinda redundant.

OR

Use `await loadingPageText.waitFor()` (https://playwright.dev/docs/api/class-locator#locator-wait-for) instead of line 27

Reply:
I think the `waitFor` seems like a good idea to have considering it returns if the entire text is visible